product/static/templates/products/user_products.js

```diff
@@ -0,0 +1,41 @@
+function product_html(data) {
+    return ('<tr id=' + data.id + '>' +
+    '<td><input class = "name" type="text" readonly value=' + data.Name + '></td> ' +
+    '<td><input class="detail" type="text" readonly value=' + data.Detail + '></td>' +
+    '<td><input class="price" type="text" readonly value=' + data.Price + '></td>' +
+    '</tr>');
+}
+function append_product(data, append) {
+    var html_string = product_html(data);
+    append == undefined ? $("#main").append(html_string) : $("#main").html(html_string);
+
+}
+
+
+function fill() {
+    if ($("#user").val()) {
+        $.ajax({
+            type: 'GET',
+            url: 'http://localhost:8000/app/my_products/' + $("#user").val() + '/',
```

Comment:
You should save the base url of the API at some global location to avoid repeating `http://localhost:8000`. 
If url of the server changes, you don not have to replace it everywhere in the code.